### ISSUE TITLE
[FR]: Streaming results format to stdout

### ISSUE THREAD

**Issue by davidmatson** [google/googletest#4665]
### Does the feature exist in the most recent commit?

No

### Why do we need this feature?

Currently, we have per-OS code for stream_result_to, and Windows support is not yet available.

### Describe the proposal.

Provide a switch to have stdout report progress in the same format used by stream_result_to rather than the human-readable format.

Perhaps:
--gtest-machine-results

### Is the feature specific to an operating system, compiler, or build system version?

No

**Comment by RedMarcher** [comment]
Hello, I'm new to this particular project. Would you mind telling me which file the current implementation for stdout is located at? You mentioned that it was different for each OS, so is the feature implementation currently spread out among multiple different files?

**Comment by RedMarcher** [comment]
Looking over your comment more closely, do you just want a feature where whatever is currently spit out of stream_result_to is also available to stdout?

**Comment by davidmatson** [comment]
> Looking over your comment more closely, do you just want a feature where whatever is currently spit out of stream_result_to is also available to stdout?

yes (and also not having the pretty-print format written stdout)

**Comment by RedMarcher** [comment]
> > Looking over your comment more closely, do you just want a feature where whatever is currently spit out of stream_result_to is also available to stdout?
> 
> yes (and also not having the pretty-print format written stdout)

So basically I should add a new flag using one of the names you proposed to activate this feature?

**Comment by davidmatson** [comment]
> So basically I should add a new flag using one of the names you proposed to activate this feature?

This is a feature request to add a new flag that would control this behavior, yes.

**Comment by davidmatson** [comment]
@derekmauro - thoughts on this proposal? We'd be interested in working on a draft implementation, if there's agreement that this would be a good idea.

**Comment by RedMarcher** [comment]
> [@derekmauro](https://github.com/derekmauro) - thoughts on this proposal? We'd be interested in working on a draft implementation, if there's agreement that this would be a good idea.

I already have a sort of merge request. Unfortunately no one has looked at it yet. This project is a little bit quiet. If you could take a look and give me your thoughts, I'd appreciate it.

https://github.com/google/googletest/pull/4668

**Comment by davidmatson** [comment]
@RedMarcher - I think we'll be best off if we can get someone from the GoogleTest team to consider the feature request before we get too deep into implementation details. Sometimes it can take a while for those folks to review, and I'm not sure of any way to accelerate that process.

**Comment by RedMarcher** [comment]
> [@RedMarcher](https://github.com/RedMarcher) - I think we'll be best off if we can get someone from the GoogleTest team to consider the feature request before we get too deep into implementation details. Sometimes it can take a while for those folks to review, and I'm not sure of any way to accelerate that process.

Alright, I'll set my pull request to draft. At least all the boiler plate is finished.
